src/Preprocessor.zig

```diff
@@ -1232,3 +1303,45 @@ fn printInBetween(slice: []const u8, w: anytype) !void {
     }
     try w.writeAll(in_between);
 }
+
+fn printDefineMap(pp: *Preprocessor) !void {
+    var kiter = pp.defines.keyIterator();
+    var k = kiter.next();
+    while (k != null) : (k = kiter.next()) {
+        const valptr = pp.defines.getPtr(k.?.*);
+        std.debug.print("{s}: [{*}]\n", .{ k.?.*, valptr });
+    }
+    std.debug.print("\n", .{});
+}
+
+fn debugTokenBuf(pp: *Preprocessor, buf: []const Token) !void {
+    _ = pp;
+    var i: u64 = 0;
+    while (i < buf.len) : (i += 1) {
+        const slice = pp.expandedSlice(buf[i]);
+        if (std.mem.eql(u8, slice, " ")) {
+            std.debug.print("⎵", .{});
+        } else if (std.mem.eql(u8, slice, "")) {
+            std.debug.print("({}) ", .{buf[i].id});
+        }else {
+            std.debug.print("{s} ", .{slice});
+        }
+    }
+    std.debug.print("[{} tokens]\n", .{buf.len});
+}
+
+fn addTestSource(pp: *Preprocessor, path: []const u8, content: []const u8) !Source {
```

Comment:
Oops, I had used this function for unit testing, it is not needed anymore, so I'll remove it.